src/lib/flexbox/api/flex.ts

```diff
@@ -77,10 +77,14 @@ export class FlexDirective extends BaseFxDirective implements OnInit, OnChanges,
   @Input('fxFlex.lt-md') set flexLtMd(val) { this._cacheInput('flexLtMd', val); };
   @Input('fxFlex.lt-lg') set flexLtLg(val) { this._cacheInput('flexLtLg', val); };
   @Input('fxFlex.lt-xl') set flexLtXl(val) { this._cacheInput('flexLtXl', val); };
-
   /* tslint:enable */
-  // Explicitly @SkipSelf on LayoutDirective and LayoutWrapDirective because we want the
-  // parent flex container for this flex item.
+
+  /**
+   * Constructor
+   *
+   * Note: Explicitly @SkipSelf on LayoutDirective and LayoutWrapDirective because we are looking
```

Comment:
This note might be better as a regular comment, otherwise it'll show up in any generated docs and typings.